industrial_ci/mockups/testpkg_broken_install/script/sample_talker.py

```diff
@@ -0,0 +1,10 @@
+# Taken from http://wiki.ros.org/rospy/Overview/Publishers%20and%20Subscribers
```

Comment:
The sample talker is not really needed for this package, but I am getting picky ;)